plan_entitlements.go

```diff
@@ -0,0 +1,157 @@
+package lago
+
+import (
+	"context"
+	"fmt"
+)
+
+type PlanEntitlementRequest struct {
+	client *Client
+}
+
+type PlanEntitlement struct {
+	Code        string                     `json:"code"`
+	Name        string                     `json:"name"`
+	Description string                     `json:"description"`
+	Privileges  []PlanEntitlementPrivilege `json:"privileges"`
+}
+
+type PlanEntitlementPrivilege struct {
+	Code      string          `json:"code"`
+	Name      string          `json:"name"`
+	ValueType string          `json:"value_type"`
```

Comment:
This is not marked as `required` in the OpenAPI.

Reply:
Thanks! updated